applications/crossbar/priv/couchdb/schemas/tasks.records.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "http://json-schema.org/draft-03/schema#",
+    "_id": "tasks.records",
+    "additionalProperties": false,
```

Comment:
That's on purpose. I don't want them to. Also I convert that json payload to CSV and store CSV only